openfisca_core/entities.py

```diff
@@ -53,6 +53,23 @@ def __call__(self, variable_name, period = None, options = [], **parameters):
         else:
             return self.simulation.calculate(variable_name, period, **parameters)
 
+    # JSON
+
+    @classmethod
+    def to_json(cls):
```

Comment:
@MattiSG: 
Some architectural question : should it be a core's class concern to convert itself to JSON so that it's served by the API ?

**Cons :**
- We put code in the core that is not related to generic operations, but a specific usecase (serving the API)

**Pros:**
- If we put this code in the API, we'll have to get many of the class attributes (here `plural`, `label`, `key`) instead of just calling the `to_json()` method. The API code will be somewhat more coupled with the structure of the class.

I'm more leaning towards putting this code in the API. @cbenz is there another part of OpenFisca, out of the API, that needs to convert entities to JSON ?